src/OSInfo.cpp

```diff
@@ -234,46 +258,103 @@ QString OSInfo::getOSVersion() {
 	           static_cast<unsigned long>(minor),
 	           static_cast<unsigned long>(bugfix),
 	           buildno ? buildno : "unknown");
+
+	return os;
+}
+#else
+QString OSInfo::getOSVersionOther() {
+#ifdef Q_OS_LINUX
+	QString os = getOSVersionLinux();
+	if (os.isEmpty()) {
+		os = getOSVersionOtherUname();
+	}
+
+	return os;
+#else
+	return getOSVersionOtherUname();
+#endif
+}
+
+QString OSInfo::getOSVersionOtherUname() {
+	QString os;
+	qDebug("OSInfo: Retrieving version using uname()");
```

Comment:
This debug-statment should be commented out. It doesn't seem like something a whole lot of devs will be interested ti know (I think)

Reply:
It only appears in the developer console.

Reply:
True that, but still it is one more message that a potential dev has to read, when trying to get something from the console. And (as written above) I don't think this information will be useful for the majority of cases. Thus I'd prefer the strategy to only log things that might be relevant to a broader dev-group to the console. If this isn't followed, I think we'll clutter the dev-console eventually, effectively rendering it (more or less) useless.